cve_bin_tool/checkers/__init__.py

```diff
@@ -39,3 +41,39 @@
     "gstreamer",
     "binutils",
 ]
+
+import re
+
+from cve_bin_tool.util import regex_find
```

Comment:
Let's have imports at the top, let's also use relative imports

```suggestion
from ..util import regex_find
```